pdbedit: add page; smbpasswd: improve comments & add `sudo` (#7545)
tldr-pages/tldr: pages/linux/smbpasswd.md

# smbpasswd

> > Add/remove a Samba user or change their SMB password.

Reviewer: What about?
```suggestion
> Add/remove a Samba user or change its password.
```

> Samba users must have an existing local Unix account.

> > More information: <https://manned.org/smbpasswd.8>.

Reviewer 2: ```suggestion
> More information: <https://manned.org/smbpasswd>.
```

Author: https://manned.org/smbpasswd.5 also exists. It not having `.8` okay?

Reviewer 2: Ahh :( sorry for this .. yep having an 8 is absolutely fine then

Reviewer 3: we should use mfrw's suggestion.

- Change the current user's SMB password:

`smbpasswd`

- Add a specified user to Samba and set password (user should already exist in system):

`sudo smbpasswd -a {{username}}`

- Modify an existing Samba user's password:

`sudo smbpasswd {{username}}`

- Delete a Samba user (use `pdbedit` instead if the Unix account has been deleted):

> `sudo smbpasswd -x {{username}}`

Reviewer: Also use long options?

Author: `smbpasswd` only provides short options.
